src/test/java/seedu/address/logic/parser/CommandParserTestUtil.java

```diff
@@ -18,7 +19,7 @@ public static void assertParseSuccess(Parser parser, String userInput, Command e
         try {
             Command command = parser.parse(userInput);
             assertEquals(expectedCommand, command);
-        } catch (ParseException pe) {
+        } catch (ParseException | CommandException pe) {
```

Comment:
Should we catch and deal with them separately?

Reply:
Yup ok! It was auto-added by the IDE. Will change